File: problems/problem02/problem02.js
function reverse(s) {
    /* fill me in */
    return "TODO";
}


function main() {
    let fs = require('fs');
    let stdin = fs.readFileSync(0).toString();
    let lines = stdin.trim().split('\n');
    let result = reverse(lines[0]);
    console.log(result);
}

main();

File: problems/problem02/problem02.cpp
#include "utility.h"

char* reverse(char* s) {
    /* fill me in */
    return 0;
}

int main() {
    int sz;
    char** a = read_strings(stdin, &sz);
    char* s = reverse(a[0]);
    printf("%s\n", s);
}
